newrelic-agent/src/main/java/com/newrelic/agent/jmx/JmxService.java

```diff
@@ -97,6 +92,7 @@ public void addJmxAttributeProcessor(JmxAttributeProcessor attributeProcessor) {
     @Override
     protected void doStart() {
         if (enabled) {
+            registerAgentMBeans();
```

Comment:
don't we want to have a specific config for this? Not just jmx as a whole

Reply:
See #62 buddy.  We want to do small incremental/progressive changes.